testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/util/DeltaLakeTestUtils.java

```diff
@@ -125,4 +125,14 @@ public static void removeS3Directory(AmazonS3 s3, String bucketName, String dire
         }
         while (listing.isTruncated());
     }
+
+    public static String getLocationForSchema(String bucketName, String externalLocation, String schemaName)
```

Comment:
nit: 
We have now:

```
S3_BUCKET=trino-ci-test
DATABRICKS_UNITY_EXTERNAL_LOCATION="databricks-unity/external"
```


Maybe we should have 

```
DATABRICKS_UNITY_EXTERNAL_LOCATION="s3://some-bucket/databricks-unity/external"
```

and don't be dependent on `S3_BUCKET` for this suite.

WDYT?

Reply:
makes sense to me. I will address this today.